frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -313,7 +311,9 @@ const LogsTabForPodName: React.FC<{ podName: string; isFailedPod: boolean }> = (
                             isDisabled={!logs}
                             onClick={() => {
                               const newTab = window.open();
-                              newTab?.document.write(`<pre>${logs}</pre>`);
+                              newTab?.document.write(
+                                `<html lang="en"><head><title>Raw logs</title></head><main role="main"><h1>Raw logs for pod ${podName} - step ${selectedContainer?.name}</h1><pre>${logs}</pre></main></html>`,
+                              );
```

Comment:
I think there has been a misunderstanding -- this should be just raw text from the API.

`https://<host>/api/kubernetes/api/v1/namespaces/opendatahub/pods/odh-dashboard-64b9549944-s66f2/log?cluster=local-cluster&container=odh-dashboard`

The Raw Link in OpenShift Console just opens it up to the URL we call to get the data.

Reply:
has been fixed here: https://issues.redhat.com/browse/RHOAIENG-2252